tutorials/AMB2018-02-B-AMR/constant/foamToExaCADict

```diff
@@ -0,0 +1,24 @@
+/*--------------------------------*- C++ -*----------------------------------*\
+  =========                 |
+  \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox
+   \\    /   O peration     | Website:  https://openfoam.org
+    \\  /    A nd           | Version:  10
+     \\/     M anipulation  |
+\*---------------------------------------------------------------------------*/
+FoamFile
+{
+    version     2.0;
+    format      ascii;
+    class       dictionary;
+    object      foamToExaCADict;
+}
+
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+
+execute     off;
```

Comment:
`foamToExaCA` works in serial cases, but not parallel. A seg fault is given in parallel after approx. 2 ms of simulation time if this tutorial is run in parallel.

Reply:
This bug will be fixed when issue #6 is addressed